### DISCUSSION TITLE
Publish pipeline doesn't build before publishing

### DISCUSSION THREAD

**Discussion by utrolig** [BetaHuhn/metadata-scraper#64] [Ideas] [answered]
I don't know why, but I can't seem to report issues.

Your publishing pipeline seems to be automated, but it doesn't look like it's actually building your package before publishing it.

**Comment by BetaHuhn** [comment, marked as answer]
> I don't know why, but I can't seem to report issues.

Normally you should be able to select bug report when creating a new issue. I manually transferred this discussion to #65
